upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -880,6 +880,43 @@ else
     echo "====> ${state_name} has been completed" | tee -a "${LOG_FILE}"
 fi
 
+state_name="UPGRADE_CERTMANAGER_CHART"
+state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
+if [[ ${state_recorded} == "0" && $(hostname) == "${PRIMARY_NODE}" ]]; then
+    echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
+    {
+        # Only remove these charts if installed
+        if helm list -n cert-manager --filter cray-certmanager | grep 'cray-certmanager$' > /dev/null 2>&1; then
```

Comment:
Needs a `-q` if you want to have the $ at the end of the grep.   
Does the --filter do anything?